extensions/default/src/Toolbar/ToolbarLayoutSelector.tsx

```diff
@@ -49,6 +65,58 @@ function LayoutSelector({ rows, columns, className, onSelection, ...rest }) {
   const onInteractionHandler = () => setIsOpen(!isOpen);
   const DropdownContent = isOpen ? OHIFLayoutSelector : null;
 
+  const commonPresets = [
+    {
+      icon: 'layout-common-1x1',
+      commandOptions: {
+        numRows: 1,
+        numCols: 1,
+      },
+    },
+    {
+      icon: 'layout-common-1x2',
+      commandOptions: {
+        numRows: 1,
+        numCols: 2,
+      },
+    },
+    {
+      icon: 'layout-common-2x2',
+      commandOptions: {
+        numRows: 2,
+        numCols: 2,
+      },
+    },
+    {
+      icon: 'layout-common-2x3',
+      commandOptions: {
+        numRows: 2,
+        numCols: 3,
+      },
+    },
+  ];
+
+  const advancedPresets = [
+    { icon: 'layout-advanced-mpr', title: 'MPR', commandOptions: { protocolId: 'mpr' } },
+    {
+      icon: 'layout-advanced-axial-primary',
+      title: 'Axial Primary',
+      commandOptions: { protocolId: 'primaryAxial' },
+    },
+    {
+      icon: 'layout-advanced-3d-four-up',
+      title: '3D four up',
+      commandOptions: { protocolId: 'fourUp' },
+    },
+    { icon: 'layout-advanced-3d-main', title: '3D main', commandOptions: { protocolId: 'main3D' } },
+    { icon: 'layout-advanced-3d-only', title: '3D only', commandOptions: { protocolId: 'only3D' } },
+    {
+      icon: 'layout-advanced-3d-primary',
+      title: '3D primary',
+      commandOptions: { protocolId: 'primary3D' },
+    },
+  ];
```

Comment:
move out of component please
add a todo that we should take a look at customizationService to check if these defaults are overwritten